src/rez/config.py

```diff
@@ -802,7 +804,7 @@ def _get_new_session_popen_args(self):
 
 class _PluginConfigs(object):
     """Lazy config loading for plugins."""
-    def __init__(self, plugin_data):
+    def __init__(self, plugin_data: typing.Dict[str, typing.Any]):
         self.__dict__['_data'] = plugin_data
```

Comment:
I'm curious why we do this `self.__dict__['_data']` accessing here. Perhaps there is a cleaner way to do this?

Reply:
I think it's to bypass `__getattr__`.